### PR TITLE
add_command_line_interface

### PR DESCRIPTION
**Pull request by bsantan**
Added command line inputs with input directory and output directory, and ensure all files exist in input directory.

### CODE REVIEW

**Comment by bsantan** [comment]
I tried to branch this off of Lucas' assign_nodes branch to not create any conflicts, I hope that worked

File: inputs.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                if k == 'nodevectors_node2vec':
+                    nodevectors_file = input_dir + '/' + fname
+                existence_dict[k] = 'true'
+
+    #Check for existence of all necessary files, error if not
+
+    #### Add exception
+    for k in existence_dict:
+        if existence_dict[k] == 'false':
+            raise Exception('Missing file in input directory: ' + k)
```
Comment (lagillenwater) [inline inputs.py:67]: Could specify which file is missing, though I don't think it's necessary for now.

**Review by lagillenwater — APPROVED** [review]
This looks good to me.

File: inputs.py (hunk 1 above)
Comment (bsantan) [reply inputs.py:67]: k will be the key above, which specifies part of the string/filename

Comment (lagillenwater) [reply inputs.py:67]: my bad. I see that now. Good thinking.

**Comment by lagillenwater** [comment]
Just looked at the other files for this branch. assign_nodes.py and interactive_assign_nodes.py don't have the most recent additions. I may not have pushed them to that branch or this branch may be based on main. Either way, I think we'll need to approve both pull requests for main to be totally updated. Does that seem right?

**Review by Nourah-Salem — APPROVED** [review]

**Comment by Nourah-Salem** [comment]
approved

**Pull request merged**